mods/pageviews.js

```diff
@@ -242,7 +237,6 @@ var validateYearMonthDay = function(rp) {
 
     // fake a timestamp in the YYYYMMDDHH format so we can reuse the validator
     var validDate = validateTimestamp(
-        ((rp.year === 'all-years') ? '2015' : rp.year) +
```

Comment:
So we still need to concatenate rp.year to the beginning of this string, otherwise validateTimestamp will always return false.